client/src/pages/private/SiteViewDetails.js

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { Page, CheckPermissions } from '../../components/generic';
+import { Box, Typography } from '@material-ui/core';
+import { scrollUp } from '../../utils';
+import store from 'store';
+import { Link } from 'react-router-dom';
+import routes from '../../constants/routes';
+
+
+export default ({ match }) => {
+  const [roles, setRoles] = useState([]);
+  const [site, setSite] = useState([]);
+  const [isLoadingUser, setLoadingUser] = useState(false);
+  const siteID = match.params.id;
+
+  const fetchUserInfo = async () => {
+    setLoadingUser(true);
+    const response = await fetch('/api/v1/user', {
+      headers: {
+        'Authorization': `Bearer ${store.get('TOKEN')}`,
+      },
+      method: 'GET',
+    });
+
+    if (response.ok) {
+      const { roles } = await response.json();
+      setLoadingUser(false);
+      setRoles(roles);
+    }
+  }
+
+  useEffect(() => {
+    fetchUserInfo();
+  }, []);
+
+  useEffect(() => {
+    const fetchDetails = async () => {
+      const response = await fetch(`/api/v1/employer-sites/${siteID}`, {
+        headers: {
+          'Authorization': `Bearer ${store.get('TOKEN')}`,
+        },
+        method: 'GET',
+      });
+
+      if (response.ok) {
+        setSite(await response.json());
+      }
+    }
+
+    fetchDetails();
+  }, [siteID]);
+
+  const fieldsLabelMap = {
+    'Site Contact': {
+      'First Name': 'siteContactFirstName',
+      'Last Name': 'siteContactLastName',
+    },
+    'Site Info': {
+      'Site Name': 'siteName',
+      'Business Name': 'registeredBusinessName',
+      'Address': 'address',
+      'Postal Code': 'postalCode',
+      'City': 'city',
+      'Phase 1 Allocation': 'earlyAdopterAllocation',
+      'Region': 'healthAuthority',
+    },
+    'Operator Info': {
```

Comment:
I think we should rename this to Operator Contact Info

Then in Site Info, we should add the operatorName field as well
We can label it "Operator" I think - this is often a business name rather than a person, which is why it's a separate field